src/components/Book.js

```diff
@@ -14,10 +14,36 @@ const Book = (props) => {
   };
 
   return (
-    <li>
-      <h3>{title}</h3>
-      <p>{author}</p>
-      <button type="button" onClick={handleClick}>Remove</button>
+    <li className="row bg-white border rounded p-5">
+      <div className="col">
+        <p className="text-category my-1">Action</p>
+        <h3 className="text-roboto m-0">{title}</h3>
+        <p className="text-roboto text-author">{author}</p>
+        <div className="d-flex align-items-center">
+          <button type="button" className="btn-white text-roboto text-author">Comments</button>
+          <div className="vertical-line" />
+          <button type="button" className="btn-white text-roboto text-author" onClick={handleClick}>Remove</button>
+          <div className="vertical-line" />
+          <button type="button" className="btn-white text-roboto text-author">Edit</button>
+        </div>
+      </div>
+      <div className="col row align-content-center">
+        <div className="col d-flex justify-content-end">
+          <div className="circular-progress" />
+        </div>
+        <div className="col row align-content-center p-0">
+          <p className="h3 m-0">64%</p>
+          <p className="text-secondary small m-0">Completed</p>
+        </div>
+      </div>
+      <div className="col-2 align-self-center p-0">
+        <div className="progress-divider" />
+      </div>
+      <div className="col align-content-center text-roboto">
+        <p className="text-uppercase text-secondary small m-0 text-chapter">Current Chapter</p>
```

Comment:
- Congratulations ㊗️ ㊗️ ㊗️ ㊗️ 
- 
![Capture](https://user-images.githubusercontent.com/63972670/208172228-6ff0e6cd-09e1-4911-bec5-b2488081bf4f.PNG)

Reply:
Thank you very much for review my PR! ;)